packages/validtion-errors/src/ValidationErrors.mdx

```diff
@@ -0,0 +1,48 @@
+---
```

Comment:
Not sure if we need a playground example for this component. I'd rather integrate this example into a complex `entry-editor` example that we have in `extensions` folder

Reply:
Will be done in a follow up PR.